### PR TITLE
created classes

### PR DESCRIPTION
**Pull request by Ade179**
Created classes and linked them together using 'require'

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    @parent_permission = parent_permission
+  end
+
+  def can_use_service?
+    if is_of_age? || @parent_permission == 'true'
+      'true'
+    else
+      'false'
+    end
+  end
```
Comment (TedLivist) [inline person.rb:18]: - [ ] There is no method called `is_of_age?`, rather what you have is `of_age?`. Kindly fix that.

- [ ] The instance variable `@parent_permission` is already either `true` or `false` so we don't need to check for it here as well.

- [ ] We can save some code here.

  ```
  def can_use_service?
    of_age? || @parent_permission
  end
  ```

File: teacher.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initializer(specialization)
+    super(name, age, parent_permission)
+    @specialization = specialization
+  end
+
+  def can_use_service?
+    'true'
```
Comment (TedLivist) [inline teacher.rb:10]: - [ ] This should be `true` (boolean), not `'true'` (string)

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Ade179,

Good job so far!
✔️ Classes are in different files based on convention
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Ade179👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Optional suggestions

- [ ] The License link in the README file is broken, I would suggest fixing it to make your README file more professional.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**